doc/userguide/rules/payload-keywords.rst

```diff
@@ -280,6 +280,8 @@ bsize
 With the ``bsize`` keyword, you can match on the length of the buffer. This adds
 precision to the content match, previously this could have been done with ``isdataat``.
 
+bsize uses an :ref:`unsigned 64-bits integer <rules-integer-keywords>`.
+
 An optional operator can be specified; if no operator is present, the operator will
 default to '='. When a relational operator is used, e.g., '<', '>' or '<>' (range),
 the bsize value will be compared using the relational operator. Ranges are inclusive.
```

Comment:
We should update this line where it says that ranges are inclusive, right?

Reply:
This should be its own ticket, and back ported.
Do you create it or should I ?

Reply:
I can do it, then :)

Reply:
https://redmine.openinfosecfoundation.org/issues/6708